tests/detect-dce-opnum/README.md

```diff
@@ -0,0 +1,7 @@
+# Description
+
+Tests the dce_opnum keyword
+
+# PCAP
+
+The pcap comes from dcerpc-dce-opnum Suricata-Verify folder.
```

Comment:
This makes me realize we probably do not need a new test then, could you please update the existing `dcerpc-dce-opnum` test with this rule and corresponding `test.yaml`?